runtime/js-compute-runtime/js-compute-builtins.cpp

```diff
@@ -280,7 +280,7 @@ const uint8_t base64DecodeTable[128] = {
   /* 64 */   nonAlphabet,           0,           1,           2,           3,           4,           5,           6,
   /* 72 */             7,           8,           9,          10,          11,          12,          13,          14,
   /* 80 */            15,          16,          17,          18,          19,          20,          21,          22,
-  /* 88 */            23,          24,          25, nonAlphabet, nonAlphabet, nonAlphabet, nonAlphabet,          63,
+  /* 88 */            23,          24,          25, nonAlphabet, nonAlphabet, nonAlphabet, nonAlphabet, nonAlphabet,
```

Comment:
This PR also includes a commit which fixes our atob implementation to throw an error if the supplied input contains an underscore